tests/kernel/run_ltp.pm

```diff
@@ -237,10 +238,41 @@ sub record_ltp_result {
         print $fh "Some test output could not be parsed: $results->{ignored_lines} lines were ignored.";
     }
 
+    $self->write_extra_test_result($export_details);
     $self->commit_result($details, $fh);
     return (0, $export_details);
 }
 
+sub write_extra_test_result {
+    my ($self, $details) = @_;
+    my $dir      = bmwqemu::result_dir();
+    my $filename = $details->{test_fqn} =~ s/:/_/gr;
+    my $result   = 'failed';
+    $result = 'passed'  if ($details->{test}->{result} eq 'PASS');
+    $result = 'skipped' if ($details->{test}->{result} eq 'CONF');
+
+    my $result_file = {
+        dents   => 0,
+        details => [{
+                _source => 'parser',
+                result  => $result,
+                text    => $filename . '.txt',
+                title   => $filename,
```

Comment:
Possibly we are missing a test suite field here. Although I don't think it matters too much.

Reply:
The test name is being used as the test suite name on the extra results tab.